src/ApiClient/models/InternalComms/InternalError.ts

```diff
@@ -136,17 +137,20 @@ export default class InternalError<T extends ErrorCode = ErrorCode> {
             origin.config.headers["Authorization"] = "*********";
         }
 
-        let debuginfo = JSON.stringify({ addon, origin, config: configOptions, stack });
+        let debuginfo = JSON.stringify({ addon, origin, config: configOptions, stack }, null, 2);
+        debuginfo = debuginfo.replace(/\\n/g, String.fromCharCode(10)); // replace with a litteral newline
         debuginfo = debuginfo.replace(
             /Basic (?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=)?/g,
             "Basic **************"
         );
-        debuginfo = debuginfo.replace(
-            /{"username":".+?","password":".+?"}/g,
-            '{"username":"*******","password":"*******"}'
-        );
-        if (CredentialsProvider.isTokenValid()) {
-            debuginfo = replaceAll(debuginfo, CredentialsProvider.token!.bearer, "**************");
+        debuginfo = debuginfo.replace(/"username":.".+?"/g, '"username": "*******"');
+        debuginfo = debuginfo.replace(/"password":.".+?"/g, '"password": "*******"');
+        if (AuthController.isTokenValid()) {
```

Comment:
Replace this with a check that checks if there is any token, invalid or not. I don't want any token ever leaked because we could be wrong about its validity (clock skew for example)